client/plots/barchart.js

```diff
@@ -204,6 +204,14 @@ export class Barchart {
 					settingsKey: 'asterisksVisible',
 					title: 'Display the asterisks'
 				})
+				inputs.push({
+					label: 'Multiple testing correction',
```

Comment:
what type of multiple testing correct is applied? Can we specify that on the portal somewhere?

Reply:
thanks added it to label